api/src/main/java/org/openmrs/module/referenceapplication/builtinreports/reports/ListOfDiagnosis.java

```diff
@@ -0,0 +1,92 @@
+package org.openmrs.module.referenceapplication.builtinreports.reports;
```

Comment:
Please include OpenMRS license header in all new files, see https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Concept.java#L1-L9

Move the classes to org.openmrs.module.referenceapplication.reports (skip the builtinreports)